### PR TITLE
docs: migrate errors code example and guide to standalone

### PR DESCRIPTION
**Pull request by bencodezen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/errors/NG3003.md [hunk 1]
```diff
@@ -53,10 +54,10 @@ The component ChildComponent is used in the template but importing it would crea
...
-    For example, using an intermediate interface that is stored in an independent file that can be imported to both dependent files without causing an import cycle.
+- Try to rearrange your dependencies to avoid the cycle.
+  For example, using an intermediate interface that is stored in an independent file that can be imported to both dependent files without causing an import cycle.
 
-*   Move the classes that reference each other into the same file, to avoid any imports between them.
-*   Convert import statements to type-only imports \(using `import type` syntax\) if the imported declarations are only used as types, as type-only imports do not contribute to cycles.
+- Move the classes that reference each other into the same file, to avoid any imports between them.
+- Convert import statements to type-only imports \(using `import type` syntax\) if the imported declarations are only used as types, as type-only imports do not contribute to cycles.
 
 @reviewed 2022-02-28
```
Comment (MarkTechson) [inline aio/content/errors/NG3003.md:63]: Can you please update the reviewed date?

**Review by MarkTechson — CHANGES_REQUESTED** [review]

File: aio/content/errors/NG3003.md [hunk 2]
```diff
@@ -53,10 +54,10 @@ The component ChildComponent is used in the template but importing it would crea
 Use this to identify how the referenced component, pipe, or directive has a dependency back to the component being compiled.
 Here are some ideas for fixing the problem:
 
-*   Try to rearrange your dependencies to avoid the cycle.
-    For example, using an intermediate interface that is stored in an independent file that can be imported to both dependent files without causing an import cycle.
+- Try to rearrange your dependencies to avoid the cycle.
```
Comment (JeanMeche) [inline aio/content/errors/NG3003.md:57]: This is likely unwanted, we only use `*` on the markdown lists

**Review by twerske — APPROVED** [review]

**Comment by bencodezen** [comment]
@MarkTechson Ready for round 2!

File: aio/content/examples/errors/cyclic-imports/child.component.ts [hunk 3]
```diff
@@ -1,7 +1,11 @@
-import { Component } from '@angular/core';
-import { ParentComponent } from './parent.component';
+import {Component} from '@angular/core';
+import {ParentComponent} from './parent.component';
 
-@Component({selector: 'app-child', template: 'The child!'})
+@Component({
+  standalone: true,
+  selector: 'app-child',
+  template: 'The child!',
```
Comment (jessicajaniuk) [inline aio/content/examples/errors/cyclic-imports/child.component.ts:7]: His name is Grogu.

**Review by jessicajaniuk — APPROVED** [review]

**Comment by bencodezen** [comment]
I think the only thing missing on this is to have @MarkTechson approve on this.

Let me know if I need to do anything else though!

**Review by MarkTechson — APPROVED** [review]
LGTM! ✅

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 763314f3009a9ea605bad182798035e12a70a1e6.

**Pull request closed without merging**